_posts/2016-05-25-gwf-workshop.md

```diff
@@ -0,0 +1,994 @@
+---
+layout: page
+title: Intermediate >> Advanced CartoDB Workshop
+category: intermediate
+date: 2016-05-25
+author: 'Ernesto Martínez'
+length: 2h
+---
+
+* Speaker: Ernesto Martínez · ernesto@cartodb.com · [@ernesmb](http://twitter.com/ernesmb)
+* May 25th 2016
+* Geospatial World Forum 2016 • Rotterdam
+
+## [http://bit.ly/link-to-workshop]()
+
+## Contents
+- [Importing datasets](#import)
+- [Getting your data ready](#dataset)
+- [Making a map](#map)
+- [Going spatial with PostGIS](#postgis)
+- [Webmapping apps with CartoDB.js](#cartodbjs)
+
```

Comment:
I'd put on the contents some estimation on how long should it take to lecture every section